vendor/plist_to_html/plist_to_html/PlistToHtml.py

```diff
@@ -13,6 +13,41 @@
 from xml.parsers.expat import ExpatError
 
 
+def get_file_content_hash(file_path):
```

Comment:
There is no way to use the utils from codechecker?

Reply:
If we want to keep plist to html as a separate tool it needs to be duplicated so there is no codechecker dependency.